indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1636,7 +1639,8 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
             for item in self._display:
                 if item.input_type == 'accommodation':
                     if item.id in data:
-                        lp.append(Paragraph(data[item.id].friendly_data.get('choice').encode('utf-8'), text_format))
+                        choice = data[item.id].friendly_data.get('choice', '').encode('utf-8')
+                        lp.append(Paragraph(choice, text_format))
                         arrival_date = data[item.id].friendly_data.get('arrival_date')
                         arrival_date = format_date(arrival_date) if arrival_date else ''
```

Comment:
ok, just checked the prod db. you either have the full set of choice+dates or an empty dict (in case the user didn't select anything). so if you use `if data.get(item.id)` instead of `if item.id in data` this check here shouldn't be necessary